### PR TITLE
created frontend method for the picture of the day.

### PR DESCRIPTION
**Pull request by LBarenschee**

### CODE REVIEW

File: frontend/src/components/Overview.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function Overview(){
```
Comment (Flooooooooooorian) [inline frontend/src/components/Overview.tsx:1]: Delete unused component.
If you need it for future features add the component in the related PR

File: frontend/src/pages/FavPicDetailsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default function FavPicDetailsPage(){
```
Comment (Flooooooooooorian) [inline frontend/src/pages/FavPicDetailsPage.tsx:1]: Delete unused component.

File: frontend/package.json [hunk 3]
```diff
@@ -10,9 +10,12 @@
     "@types/node": "^16.11.36",
     "@types/react": "^18.0.9",
     "@types/react-dom": "^18.0.4",
+    "axios": "^0.27.2",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "toastify": "^2.0.1",
```
Comment (Flooooooooooorian) [inline frontend/package.json:18]: remove unused dependencys or use it 😉

File: frontend/src/hooks/useNasaPicture.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import {useEffect, useState} from "react";
+import {getPictureOfTheDay} from "../service/api-service";
+
```
Comment (wehnerts) [inline frontend/src/hooks/useNasaPicture.ts:3]: Evtl nur eine Leerzeile zwischen den abschnitten?

**Review by Flooooooooooorian — APPROVED** [review]
Remove node_modules, package.json and package-lock.json from root directory

Your CI is failing...!

**Pull request merged**